woof-code/rootfs-skeleton/usr/sbin/sfs_load

```diff
@@ -1756,7 +1756,7 @@ case "$PUPMODE" in
 		else SFSMODE="tmp"
 		fi
 		;;
-	12|13) SFSMODE="y";;
+	6|12|13) SFSMODE="y";;
```

Comment:
TODO: get sfs_load to work; the whole point in the 2/6/13 distinction is allowing sfs_load to work in 6